source/coffee/test/lib/css-url-versioner.coffee

```diff
@@ -77,200 +77,72 @@ describe('cssUrl', () ->
 		)
 		return
 	)
-	
+
+	assertVersioned = (arr, indx) ->
+		return () ->
+			instance = cssVersioner({
+				content: arr[indx * 2]
+			})
+
+			it(arr[indx * 2] + ' should be convert to: ' + arr[indx * 2 + 1] + '.', () ->
+				instance.output.should.be.equal(arr[indx * 2 + 1])
+			)
 
 	describe('Generated versions', () ->
 
 		queryString = '?v=' + version
 
-		withoutQuotes = ['url(sprite.png)', 'url(sprite.png' + queryString + ')',
-						 'url(fonts/new.eot#ie)', 'url(fonts/new.eot' + queryString + '#ie)',
-						 'url(img/abc.dfg.png)', 'url(img/abc.dfg.png' + queryString + ')',
-						 'url(img/klm.nop.png#slug)', 'url(img/klm.nop.png' + queryString + '#slug)']
+		withoutQuotes = [
+			'url(sprite.png)', 'url(sprite.png' + queryString + ')',
+			'url(fonts/new.eot#ie)', 'url(fonts/new.eot' + queryString + '#ie)',
+			'url(img/abc.dfg.png)', 'url(img/abc.dfg.png' + queryString + ')',
+			'url(img/klm.nop.png#slug)', 'url(img/klm.nop.png' + queryString + '#slug)',
+			'url(file with space.woff)', 'url(file with space.woff' + queryString + ')'
+		]
 		
 		describe('Without quotes:', () ->
 
-			describe(withoutQuotes[0], () ->
-
-				instance = cssVersioner({
-					content: withoutQuotes[0]
-				})
-
-				it(withoutQuotes[0] + ' should be convert to: ' + withoutQuotes[1] + '.', () ->
-					instance.output.should.be.equal(withoutQuotes[1])
-					return
-				)
-				return
-			)
+			for i in [0..withoutQuotes.length / 2 - 1]
 
-			describe(withoutQuotes[2], () ->
-
-				instance = cssVersioner({
-					content: withoutQuotes[2]
-				})
-
-				it(withoutQuotes[2] + ' should be convert to: ' + withoutQuotes[3] + '.', () ->
-					instance.output.should.be.equal(withoutQuotes[3])
-					return
-				)
-				return
-			)
-
-			describe(withoutQuotes[4], () ->
-
-				instance = cssVersioner({
-					content: withoutQuotes[4]
-				})
-
-				it(withoutQuotes[4] + ' should be convert to: ' + withoutQuotes[5] + '.', () ->
-					instance.output.should.be.equal(withoutQuotes[5])
-					return
-				)
-				return
-			)
-
-			describe(withoutQuotes[6], () ->
-
-				instance = cssVersioner({
-					content: withoutQuotes[6]
-				})
-
-				it(withoutQuotes[6] + ' should be convert to: ' + withoutQuotes[7] + '.', () ->
-					instance.output.should.be.equal(withoutQuotes[7])
-					return
-				)
-				return
-			)
+				describe(withoutQuotes[i], assertVersioned(withoutQuotes, i))
```

Comment:
fixed in https://github.com/jansanchez/css-url-versioner/commit/2d74d01ebae9e04d0c6ddea85e3afe3e63471a37